recipes/implot/all/conanfile.py

```diff
@@ -24,41 +27,45 @@ class ImplotConan(ConanFile):
         "fPIC": True
     }
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)
 
     def config_options(self):
         if self.settings.os == "Windows":
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
```

Comment:
Technically its not necessary to use `rm_safe` here as there is an implicit contract where `config_options` will be executed before `configure`. (So if config_options deletes fPIC, then configure using rm_safe will be a 'safe' no-op).

Reply:
That's good to know. Thanks!

Reply:
Updated in [6680a2a](https://github.com/conan-io/conan-center-index/pull/16847/commits/6680a2a59c4669409640ab1a3eb01a2f46820bd2)

Reply:
I usually put it for all of them so I dont have to use my brain :)